### PR TITLE
[WIP] cli.js show helpful message

### PR DESCRIPTION
**Pull request by chocnut**
Instead of showing these error message.
screen shot 2017-12-13 at 8 41 22 pm
screen shot 2017-12-13 at 8 41 35 pm

Show them a helpful message.
screen shot 2017-12-13 at 8 49 35 pm
screen shot 2017-12-13 at 8 49 51 pm

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -73,13 +73,32 @@ program.on('--help', function() {
   console.log('');
 });
 
-// Make serve the default command
 var args = process.argv;
-if (!args[2] || !program.commands.some(c => c.name() === args[2])) {
+if (!args[2]) {
```
Comment (yeskunall) [inline bin/cli.js]: What happens if I just do `parcel build`? Won't this display a warning message specific to the `serve` command?

File: bin/cli.js [hunk 2]
```diff
@@ -73,13 +73,32 @@ program.on('--help', function() {
...
+  // no entry point.
+  displayWarningMessage();
+} else if (args[2] && args[2] === 'serve') {
```
Comment (chocnut) [inline bin/cli.js]: @yeskunall good point. I think that checking should be here as well.

File: bin/cli.js [hunk 3]
```diff
@@ -76,21 +75,21 @@ program.on('--help', function() {
 var args = process.argv;
 if (!args[2]) {
   // no entry point.
-  displayWarningMessage();
-} else if (args[2] && args[2] === 'serve') {
-  // serve but with no entry point.
-  displayWarningMessage('serve');
+  showInstructions();
+} else if (args[2] === 'serve' || args[2] === 'build') {
```
Comment (yeskunall) [inline bin/cli.js]: This is still not fully correct. `build` doesn't require you to provide an entry point.

Also, you need to remove the first `if` altogether. It makes no sense.

**Comment by devongovett** [comment]
An entry point is actually not required: if not passed, the `main` field from `package.json` will be used instead.

**Pull request closed without merging**